cstrml/market/src/lib.rs

```diff
@@ -228,10 +228,10 @@ decl_storage! {
         map hasher(twox_64_concat) T::Hash => Option<StorageOrder<T::AccountId, BalanceOf<T>>>;
 
         /// Order status counts used for punishment
-        pub ProviderPunishments get(fn provider_punishments):
-        map hasher(twox_64_concat) T::Hash => Option<ProviderPunishment<BalanceOf<T>>>;
+        pub MerchantPunishments get(fn merchant_punishments):
```

Comment:
should we write on_inialize to transform the storage？

Reply:
no, we are not supposed to do forkless upgrade between m1 to m2.